pkg/helpers/legalnotice/legalnotice.go

```diff
@@ -0,0 +1,42 @@
+package legalnotice
```

Comment:
Call it just notice, motd, or welcome screen, or something like that. Not everywhere it will be used as legalnotice.